src/chicken-parmigiana.md

```diff
@@ -0,0 +1,54 @@
+# Chicken Parmigiana
+
+Staple Australian pub food (aka Chicken Parma).
+
+![Chicken Parmigiana](pix/chicken-parmigiana.webp)
+
+- Prep time: 10 min
+- Cook time: 30 min
+- Servings: 2
+
+## Ingredients
+
+- 1 boneless/skinless chicken breast
+- 1 cup plain flour
+- 2 eggs
+- 1 cup breadcumbs
+- 1/2 cup grated parmesan cheese
+- italian mixed herbs
+- several tablespoons olive oil
```

Comment:
elaborate?

Reply:
Fixed up spelling mistake here, made it look a bit neater